src/peer/negotiator.js

```diff
@@ -114,6 +114,23 @@ class Negotiator extends EventEmitter {
         }
       });
 
+      if (this._pc.getSenders().every(sender => {
```

Comment:
About L:117-130, what happens in the use case described below?

1. The user has both an audio and a video track; which means, _pc.getSenders() should return the array including 2 rtpSender.
2. Then user wants to stop sending the video track to reduce bandwidth, so s/he calls `replaceStream(newStream)` where `newStream` contains only the audio track.
3. Does the user can successfully create an offer SDP and communicate with the other party after negotiation as the user expects?